app/models/user.rb

```diff
@@ -153,9 +153,7 @@ def pending_refund_payments
       projects: {
         state: 'failed'
       },
-      state: 'paid',
-      gateway: 'Pagarme',
```

Comment:
I believe there are other parts of the system that expect this payment to be done through pagarme. The best approach here is to split this scope in 2.